backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java

```diff
@@ -115,16 +116,16 @@ public ISubscriptionManager getSubscriptionManager() {
             if (formInput instanceof UpdateWidgetFocusInput) {
                 UpdateWidgetFocusInput input = (UpdateWidgetFocusInput) formInput;
                 this.widgetSubscriptionManager.handle(input);
-                result = Optional.of(new EventHandlerResponse(false, representation -> false, new UpdateWidgetFocusSuccessPayload(input.getWidgetId())));
+                result = Optional.of(new EventHandlerResponse(ChangeKind.NOTHING, new UpdateWidgetFocusSuccessPayload(input.getWidgetId())));
```

Comment:
It seems strange to say that "NOTHING" changed here, and return a payload saying the opposite ("a widget focus was updated", so *something* changed). I understand what's really happenning, it's just the wording.

Reply:
Yeah, that's odd but I couldn't fine a proper change kind for that but a "focus updated" change kind could be useful including for other representations.